tf-ansible/README.md

```diff
@@ -0,0 +1,57 @@
+# Overview
+
+The blueprints in this directory demonstrate the use of Terraform to provision an application (NGINX) running on a deployed server in the cloud platform of your choice. Terraform deploys the necessary resources (VPCs, networking, instances, etc.), and Ansible configures the instances once they have been created.
+
+# Blueprints
+
+The following example blueprints are provided:
+
+* [aws.yaml](./aws.yaml)
+* [gcp.yaml](./gcp.yaml)
+* [azure.yaml](./azure.yaml)
+
+The blueprints are very similar, with the exception of the Terraform used to deploy to the appropriate cloud provider and any corresponding credentials used by Terraform.
+
+# Usage
+
+To get started with these blueprints:
+
+1. Download the tf-ansible example from the [releases](https://github.com/cloudify-community/blueprint-examples/releases) page.
+     * This step is optional. You may also choose to point your Cloudify Manager to the .zip file hosted on GitHub when uploading the blueprint.
+2. Upload the blueprint package to your Cloudify manager, specifying the appropriate blueprint for your deployment (e.g, AWS, GCP, or Azure). For example, to deploy the AWS blueprint using the Cloudify CLI:
+```
+cfy blueprint upload -b tf-ansible-aws -n aws.yaml tf-ansible.zip
+```
+3. Ensure that any prerequisites are met for the cloud provider that you will use (see below)
+
+## Prerequisites
+
+All blueprints require a `private_key_content` secret with a private key that Ansible will use to connect to the deployed hosts. This should be the private key for the corresponding public key that is specified via the `public_key_content` deployment input. For example, to upload a private key using the CLI:
+
+```
+cfy secrets create -uf .ssh/id_rsa private_key_content
+```
+
+### AWS
+
+The AWS blueprint example requires the following secrets to be configured in your Cloudify manager:
+
+* `aws_access_key_id` - An AWS Access Key ID with sufficient permissions to deploy all resources via Terraform
+* `aws_secret_access_key` - The corresponding AWS Secret Access Key with sufficient permissions to deploy all resources via Terraform
+
+### GCP
+
+The GCP blueprint example requires the following secrets to be configured in your Cloudify manager:
+
+* `gcp_credentials_json` - The GCP credentials JSON file, in string format. This can be easily generated by [creating a service account](https://cloud.google.com/docs/authentication/getting-started), downloading the JSON file, and running the following command to convert the JSON file to a string: `jq -c . /path/to/gce-creds.json`
```

Comment:
I prefer to use the name `gcp_credentials`, so this will be consistent across examples. Also a good idea to include the command `cfy secrets create gcp_credentials -f /path/to/gce-creds.json`